src/smart-components/group/role/group-roles.js

```diff
@@ -166,6 +168,7 @@ const GroupRoles = ({
               className: 'pf-m-hidden-on-md',
             },
             onClick: () => {
+              history.push(`/groups/detail/${uuid}/roles/add_roles`);
               (e) => disableAddRoles && e.preventDefault();
```

Comment:
@karelhala @Hyperkid123 Does this bit do anything?

Reply:
It should. Is it not changing the location?